packages/legacy-scripting-runner/zfactory.js

```diff
@@ -24,8 +24,12 @@ const convertBundleRequest = (bundleOrBundleRequest) => {
     };
   }
 
-  bundleRequest.qs = bundleRequest.params || {};
-  bundleRequest.body = bundleRequest.data || '';
+  if (!bundleRequest.qs && bundleRequest.params) {
+    bundleRequest.qs = bundleRequest.params;
+  }
+  if (!bundleRequest.body && bundleRequest.data) {
+    bundleRequest.body = bundleRequest.data;
+  }
```

Comment:
Borrowed this from the [code](https://github.com/zapier/lambda-wrapper/blob/aeb3fc53fe07ad0990fe1fb6b3daea7461c623a3/lambda-handlers/evalbox-javascript/utils.js#L236-L240.) we used for WB.